src/main/scala/mesosphere/marathon/io/IO.scala

```diff
@@ -85,6 +85,15 @@ object IO {
     new String(out.toByteArray, "UTF-8")
   }
 
+  def withResource[T](path: String)(fn: InputStream => T): Option[T] = {
```

Comment:
What do you mean with path sanitisation? 
The method here (in the IO class) does not prevent anything.
In the WebJar case, you have only access to /META-INF/resources/webjars/*